<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchWidget</class>
 <widget class="QWidget" name="SearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>51</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QPushButton" name="pbClose">
     <property name="text">
      <string>Clo&amp;se</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lSearch">
     <property name="text">
      <string>Find: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cbSearch">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <property name="maxCount">
      <number>100</number>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::NoInsert</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbPrevious">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&amp;Previous</string>
     </property>
     <property name="icon">
      <iconset resource="../icons.qrc">
       <normaloff>:/icons/prev.png</normaloff>:/icons/prev.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbNext">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&amp;Next</string>
     </property>
     <property name="icon">
      <iconset resource="../icons.qrc">
       <normaloff>:/icons/next.png</normaloff>:/icons/next.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbCase">
     <property name="text">
      <string>&amp;Match case</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbClose</sender>
   <signal>clicked()</signal>
   <receiver>SearchWidget</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbNext</sender>
   <signal>clicked()</signal>
   <receiver>SearchWidget</receiver>
   <slot>findNext()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbPrevious</sender>
   <signal>clicked()</signal>
   <receiver>SearchWidget</receiver>
   <slot>findPrevious()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>551</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>519</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>findNext()</slot>
  <slot>findPrevious()</slot>
 </slots>
</ui>
